packages/console/src/hooks/use-swr-options.ts

```diff
@@ -0,0 +1,19 @@
+import { SWRConfiguration } from 'swr';
+
+import useSwrFetcher from './use-swr-fetcher';
+
+const useSwrOptions = (): SWRConfiguration => {
+  const fetcher = useSwrFetcher();
+
+  return {
+    fetcher,
+    onErrorRetry: (error) => {
+      if (error.status === 401 || error.status === 403) {
+        // eslint-disable-next-line no-useless-return
+        return;
```

Comment:
![image](https://user-images.githubusercontent.com/14722250/172308969-37706f2f-f22f-4322-8898-691883300524.png)

Reply:
per doc you mentioned:
1. setting this will disable the default retry strategy, swr will NOT retry at all
2. should use `shouldRetryOnError` if it accepts a function, otherwise let's figure out something else

Reply:
and, that's exact the point of this lint rule. double check your function, there's no difference comparing to `() => undefined`